src/assets/scripts/sections/product.js

```diff
@@ -79,6 +83,33 @@ export default register('product', {
         this.updateImages.bind(this),
       );
     }
+
+    this.initImageSwitch();
+  },
+
+  initImageSwitch() {
+    if (!$(selectors.productThumbs).length) {
+      return;
+    }
+
+    $(selectors.productThumbs, this.$container)
+      .on('click', (evt) => {
+        evt.preventDefault();
+        const imageId = $(evt.currentTarget).data('thumbnail-id');
+        this.switchImage(imageId);
+        this.setActiveThumbnail(imageId);
+      })
+      .on('keyup', this.handleImageFocus.bind(this));
+  },
+
+  handleImageFocus(evt) {
+    if (evt.keyCode !== keyboardKeys.ENTER) {
+      return;
+    }
+
+    $(selectors.productFeaturedImage, this.$container)
```

Comment:
I think you can use `this.$featuredImage` that you define in `onLoad()` ?

Reply:
Thanks for the good eyes, I did not notice it.